src/clusterfuzz/_internal/bot/tasks/utasks/regression_task.py

```diff
@@ -527,6 +642,8 @@ def handle_regression_build_setup_error(output: uworker_msg_pb2.Output):  # pyli
       wait_time=build_fail_wait)
 
 
+# TODO: Delete this once all uworkers are past
```

Comment:
I'm not sure what this means, and can you make an issue for this?

Reply:
Ah yes, forgot about cleaning this up. Will do.

Reply:
Done.